### PR TITLE
support the fp16 mode of lite engine, test=develop

### PR DESCRIPTION
**Pull request by Shixiaowei02**
### PR types

New features

### PR changes

Others

### Describe

support the fp16 mode of lite engine, test=develop

### CODE REVIEW

File: paddle/fluid/inference/lite/tensor_utils.cc [hunk 1]
```diff
@@ -78,6 +78,8 @@ PrecisionType GetLitePrecisionType(framework::proto::VarType::Type type) {
       return PrecisionType::kInt32;
     case framework::proto::VarType_Type_INT64:
       return PrecisionType::kInt64;
+    case framework::proto::VarType_Type_FP16:
+      return PrecisionType::kFP16;
     default:
       PADDLE_THROW(platform::errors::Unimplemented(
           "Unsupported precision type. Now only supports FP32, INT8, INT32 and "
```
Comment (jiweibo) [inline paddle/fluid/inference/lite/tensor_utils.cc]: 之后的Pr添加上half？

**Review by jiweibo — DISMISSED** [review]
LGTM

File: paddle/fluid/inference/lite/tensor_utils.cc (hunk 1 above)
Comment (Shixiaowei02) [reply paddle/fluid/inference/lite/tensor_utils.cc]: done

**Review by jiweibo — APPROVED** [review]
LGTM

**Pull request closed without merging**